lib/smart_answer_flows/marriage-abroad/outcomes/_synonyms_of_cp.govspeak.erb

```diff
@@ -114,7 +114,7 @@
   - ’domestic partnership’ (in Nova Scotia)
   - ’union civile’, or ’civil union’ (in Quebec)
 <% when 'new-zealand' %>
-  You may be able to get married or enter into a ‘civil union’ at the British embassy or consulate in New Zealand. Civil unions are recognised as civil partnerships under British law.
+  Civil unions are recognised as civil partnerships under British law.
```

Comment:
Minor: It would really nice if the commit note could explain why this sentence is being removed e.g. is it because you can no longer get married at the British embassy, or is it because we no longer want to mention it as a possibility, or something other reason? I know you may not have this information from the Content Team and they in turn might not have it from the FCO, but it would be really good to get them to start providing this kind of information.